fili-core/src/main/java/com/yahoo/bard/webservice/data/dimension/impl/LuceneSearchProvider.java

```diff
@@ -739,7 +739,7 @@ private Query getFilterQuery(Set<ApiFilter> filters) {
         } finally {
             lock.readLock().unlock();
         }
-        return new SinglePagePagination<>(
+        return new SinglePagePagination<DimensionRow>(
```

Comment:
Diamond operator seems to be fine (I see method signature already specified type)

Reply:
I fixed this and then accidentally reverted it.